src/app/interactions/drag-drop/kanban-sample/kanban-sample.component.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-shadow */
 /* eslint-disable @typescript-eslint/naming-convention */
-import { ChangeDetectorRef, Component, ElementRef, OnInit, Renderer2, ViewChild } from '@angular/core';
+import { ChangeDetectorRef, Component, ElementRef, OnInit, Renderer2, viewChild, ViewChild } from '@angular/core';
```

Comment:
```suggestion
import { ChangeDetectorRef, Component, ElementRef, OnInit, Renderer2, ViewChild } from '@angular/core';
```

Reply:
Agree with Deyan. Why is this imported with lower case too?

Reply:
probably imported by mistake due to good intellisense and I missed it. Removed it now.